Source/CovPassApp/Source/Resources/Locale/en.lproj/Localizable.strings

```diff
@@ -11,226 +11,226 @@
+"next_onboarding_page_button_title" = "Next";
 
-"vaccination_first_onboarding_page_title" = "Erster Schritt: Übertragen Sie das COVID-Zertifikat auf Ihr Smartphone";
+"vaccination_first_onboarding_page_title" = "First step: Transfer the COVID certificate to your smartphone";
 
-"vaccination_first_onboarding_page_message" = "Scannen Sie mit der CovPass-App das EU Digitale COVID-Zertifikat für eine Corona-Impfung, ein negatives Corona-Testergebnis oder eine Genesung von der Corona-Infektion. Halten Sie dafür die Kamera Ihres Smartphones über den QR-Code. Das COVID-Zertifikat wird sofort auf Ihr Smartphone übertragen.";
+"vaccination_first_onboarding_page_message" = "Scan the EU digital COVID vaccination certificate, a negative COVID test result or recovery from a COVID infection with the CovPass app. To do so, hold the camera of your smartphone over the QR code. The COVID certificate is immediately transferred to your smartphone.";
 
-"vaccination_second_onboarding_page_title" = "Zweiter Schritt: Rufen Sie den QR-Code in der CovPass-App auf";
+"vaccination_second_onboarding_page_title" = "Second step: Open the QR code in the CovPass app";
 
-"vaccination_second_onboarding_page_message" = "Mit dem QR-Code können Sie eine Corona-Impfung, ein negatives Corona-Testergebnis oder eine Genesung von einer Corona-Infektion nachweisen. Nach dem Scannen wird der QR-Code direkt in der CovPass-App angezeigt. Bitte beachten Sie, dass eine ausreichende Wirksamkeit des Impfschutzes erst etwa 14 Tage nach der letzten Impfung eintritt. Der QR-Code für den vollständigen Impfschutz ist deshalb erst nach Ablauf der 14 Tage gültig.";
+"vaccination_second_onboarding_page_message" = "With the QR code you can prove a COVID vaccination, a negative COVID test result or recovery from a COVID infection. After scanning, the QR code is displayed directly in the CovPass app. Please note that sufficient effectiveness of the vaccination protection only occurs about 14 days after the last vaccination. The QR code for full vaccination protection is therefore only valid after the 14 days have elapsed.";
 
-"vaccination_third_onboarding_page_title" = "Dritter Schritt: Zeigen Sie den QR-Code bei Bedarf vor";
+"vaccination_third_onboarding_page_title" = "Third step: Show the QR code if required";
 
-"vaccination_third_onboarding_page_message" = "Bei Bedarf können Sie den QR-Code in der CovPass-App vorzeigen. Der QR-Code wird dann von der prüfenden Person gescannt. Halten Sie dazu ergänzend Ihr Ausweisdokument bereit. Da der QR-Code nur auf Ihrem Smartphone gespeichert ist, können Sie ihn jederzeit auch ohne Internetverbindung anzeigen lassen.";
+"vaccination_third_onboarding_page_message" = "If required, you can show the QR code in the CovPass app. The QR code will then be scanned by the examiner. Please also have your identity document ready. As the QR code is only stored on your smartphone, you can display it at any time even without an internet connection.";
```

Comment:
"," after "any time" and before "even"?

```suggestion
"vaccination_third_onboarding_page_message" = "If required, you can show the QR code in the CovPass app. The QR code will then be scanned by the examiner. Please also have your identity document ready. As the QR code is only stored on your smartphone, you can display it at any time, even without an internet connection.";
```

Reply:
I guess it's optional, but it surely makes sense here.